cider/exec/nextgen/jitlib/llvmjit/LLVMJITFunction.cpp

```diff
@@ -263,6 +264,45 @@ JITValuePointer LLVMJITFunction::getArgument(size_t index) {
   }
 }
 
+JITValuePointer LLVMJITFunction::packJITValuesImpl(
+    const std::vector<JITValuePointer>& vals,
+    const uint64_t alignment) {
+  int64_t memory_count = 0;
+  // record memory index address
+  std::vector<int64_t> memory_index;
+  for (auto val : vals) {
+    memory_index.push_back(memory_count);
+    memory_count += getJITTypeSize(val->getValueTypeTag());
+    // memory align
+    memory_count = (memory_count + alignment) & ~(alignment - 1);
+  }
+
+  llvm::AllocaInst* allocated_memory = ir_builder_->CreateAlloca(
```

Comment:
memory leak risk?

Reply:
Yeah，this memory is allocated on the stack，so there is no such risk

Reply:
I guess you may pack each row in join scenario. If so, stack may overflow?

Reply:
Actually I will use to pack mutiple column data in one row，the stack may be 8MB i'm not sure. So it is unlikely that this overflow. And I've push these code in this PR #226